From e19abfd7926798751a6edeefe406da0415b3b5f6 Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 7 Mar 2014 17:45:38 +0000
Subject: [PATCH 0525/1077] MANTIS 0029380 change EventManager: Events are not
 sent as specified in CUSS Spec

added an interface method in PlatformIF

git-svn-id: svn://localhost/SelfServiceCommon/trunk@655 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index fbd2e8f8..a7be53a1 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "1.6";
+    const string interface_version = "1.7";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -315,6 +315,8 @@ module PlatformManagement
             @param componentType - type of the component that received the data; e.g.: BCR for barcode reader
             @param data - data as defined in cuss types.idl(msgDataType, aeaDataType, svgDataType, ...) */
         void dataReceived(in string componentType, in any data);
+        
+        void setupVCompListener(in string appRef, in string compRef, in any action, in any eventAcquireFilter, in any eventListener, in any elud);
     };
 };
 
-- 
2.41.0.windows.1

